client-device-connection/src/main/java/org/eclipse/hono/deviceconnection/common/CacheBasedDeviceConnectionInfo.java

```diff
@@ -105,7 +105,7 @@ public CacheBasedDeviceConnectionInfo(final Cache<String, String> cache, final T
      * @throws NullPointerException if cache or tracer is {@code null}.
      */
     public CacheBasedDeviceConnectionInfo(final Cache<String, String> cache, final Tracer tracer,
-            final AdapterInstanceStatusProvider adapterInstanceStatusProvider) {
+                                          final AdapterInstanceStatusProvider adapterInstanceStatusProvider) {
```

Comment:
does this indentation come from the formatting rules from the `eclipse` folder? If not, please revert it. Adding many formatting changes makes it hard to see the *real* changes when reviewing PRs ...